### PR TITLE
Update networks doc to avoid passive voice & reduce word count

### PR DESCRIPTION
**Pull request by setgree**
* The lead sentence in this page should define networks rather than explaining something about how they work, so I switched the first and second sentences around
* Some passive voice, e.g. "Agreement on transactions...is decided by a consensus of peers," which I rearranged to put the actors first 
* some overlong, repetitive sentences that I condensed

### Further suggestions
* The page has a few more instances of passive voice whose intent I wasn't clear enough about to feel confident editing, e.g. "it's useful to know" -- perhaps a real expert can take a second look 😃 
* Perhaps the deprecated networks can be moved to the bottom of the page under a header 'Deprecated networks?' I wasn't sure enough about this to make the change

Thanks in advance!

### CODE REVIEW

File: src/content/developers/docs/networks/index.md [hunk 1]
```diff
@@ -27,15 +27,15 @@ When people and exchanges discuss ETH prices, they're talking about Mainnet ETH.
 
 In addition to Mainnet, there are public testnets. These are networks used by protocol developers or smart contract developers to test both protocol upgrades as well as potential smart contracts in a production-like environment before deployment to Mainnet. Think of this as an analog to production versus staging servers.
 
-It’s generally important to test any contract code you write on a testnet before deploying to the Mainnet. If you're building a dapp that integrates with existing smart contracts, most projects have copies deployed to testnets that you can interact with.
+We advise you to test any contract code you write on a testnet before deploying to the Mainnet. Among dapps that integrate with existing smart contracts, most projects have copies deployed to testnets.
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:30]:
```suggestion
You should test any contract code you write on a testnet before deploying to Mainnet. Among dapps that integrate with existing smart contracts, most projects have copies deployed to testnets.
```

File: src/content/developers/docs/networks/index.md [hunk 2]
```diff
@@ -27,15 +27,15 @@ When people and exchanges discuss ETH prices, they're talking about Mainnet ETH.
...
 
-Most testnets use a proof-of-authority consensus mechanism. This means a small number of nodes are chosen to validate transactions and create new blocks – staking their identity in the process. It's hard to incentivise mining on a proof-of-work testnet which can leave it vulnerable.
+Most testnets use a proof-of-authority consensus mechanism. This means a small number of nodes are chosen to validate transactions and create new blocks – staking their identity in the process. Testnets do not incentivize proof-of-work mining, which can leave them vulnerable.
 
 ETH on testnets has no real value; therefore, there are no markets for testnet ETH. Since you need ETH to actually interact with Ethereum, most people get testnet ETH from faucets. Most faucets are webapps where you can input an address which you request ETH to be sent to.
 
 #### Sepolia {#sepolia}
 
-A proof-of-work testnet; this means it's the best like-for-like representation of Ethereum.
+A proof-of-work testnet that has the best available like-for-like representation of Ethereum.
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:38]:
```suggestion
A proof-of-work testnet—the best like-for-like representation of Ethereum Mainnet.
```

File: src/content/developers/docs/networks/index.md [hunk 3]
```diff
@@ -133,9 +133,9 @@ There are projects and tools dedicated to assist with this. Learn more about [de
 
 ### Consortium networks {#consortium-networks}
 
-The consensus process is controlled by a pre-defined set of nodes that are trusted. For example, a private network of known academic institutions that each govern a single node, and blocks are validated by a threshold of signatories within the network.
+The consensus process is controlled by a pre-defined set of nodes that are trusted -- for example, a private network of known academic institutions that each govern a single node, and blocks are validated by a threshold of signatories within the network.
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:136]:
```suggestion
The consensus process is controlled by a pre-defined set of nodes that are trusted. For example, a private network of known academic institutions that each govern a single node, and blocks are validated by a threshold of signatories within the network.
```

File: src/content/developers/docs/networks/index.md (hunk 1 above)
Comment (setgree) [reply src/content/developers/docs/networks/index.md:30]: yours  is better 👍

File: src/content/developers/docs/networks/index.md [hunk 4]
```diff
@@ -5,17 +5,17 @@ lang: en
...
 
 ## Prerequisites {#prerequisites}
 
-You should understand the basics of Ethereum before reading up on the different networks as the test networks will give you a cheap, safe version of Ethereum to play around with. Try our [introduction to Ethereum](/developers/docs/intro-to-ethereum/).
+You should understand the [basics of Ethereum](/developers/docs/intro-to-ethereum/) before reading up on the different networks, as the test networks will give you a cheap, safe version of Ethereum to play around with.
 
 ## Public networks {#public-networks}
 
-Public networks are accessible to anyone in the world with an internet connection. Anyone can read or create transactions on a public blockchain and validate the transactions being executed. Agreement on transactions and the state of the network is decided by a consensus of peers.
+Public networks are accessible to anyone in the world with an internet connection. Anyone can read or create transactions on a public blockchain and validate the transactions being executed. A consensus of peers decides on agreement of transactions and the state of the network.
```
Comment (minimalsm) [inline src/content/developers/docs/networks/index.md:18]:
```suggestion
Public networks are accessible to anyone in the world with an internet connection. Anyone can read or create transactions on a public blockchain and validate the transactions being executed. The consensus among peers decides on the inclusion of transactions and the state of the network.
```

**Comment by minimalsm** [comment]
> The page has a few more instances of passive voice whose intent I wasn't clear enough about to feel confident editing, e.g. "it's useful to know" -- perhaps a real expert can take a second look 😃

I think the tone of that sentence is fine. What would you suggest?

> Perhaps the deprecated networks can be moved to the bottom of the page under a header 'Deprecated networks?' I wasn't sure enough about this to make the change

Ideally, yes. However, on Ethereum things being deprecated and things being unused are two different stories 😅 e.g. Rinkeby is still the most widely used testnet for development purposes right now. It isn't ideal, but keeping them together (for the list of faucets) would be my preference for now. Hopefully we can remove them soon :-)

**Comment by setgree** [comment]
👋 thanks for the quick turnaround! I observe that you and I part ways on the question of sentence fragments, but I happily defer. (A matter of taste! 😃)
* RE: "it's useful to know" -- I'm not sure TBH, perhaps passive voice is the right thing here   
* RE: deprecated vs. out of use -- totally see your point, I lacked that context

Until next time,

**Pull request merged**